getting-started/CONCEPTS/STREAMING.md

```diff
@@ -208,6 +210,74 @@ planner=ActionPlanner(
 
 ```
 
+## Custom `StreamingResponse` Management
+
+### Definitions
+
+- **Stream ID**: A unique identifier for the stream, assigned after the initial update is sent. It ensures that all subsequent messages are associated with the correct stream.
+- **Sequence Numbers**: Each message in the stream is assigned a sequence number (`streamSequence`), starting from 1 and incrementing with each message. This helps maintain the order of messages.
+
+
+The class also maintains an internal queue for outgoing activities to ensure they are sent in order and without overloading the Teams client. 
+
+### Implementation
+
+#### 1. Initialize `StreamingResponse`
+
+To start using the `StreamingResponse` class, you need to create an instance of it within your bot's turn handler. 
+
+   ```javascript
+   const streamingResponse = new StreamingResponse(context);
+   ```
+
+
+#### 2. Send Informative Updates
+
+   ```javascript
+   streamingResponse.queueInformativeUpdate('Searching for the information you requested...');
+   ```
+
+#### 3. Queuing Text Chunks
+
+As your bot generates or retrieves content, you can send partial messages to the user.
+
+**Note**: It is important to include a delay between the chunks to avoid spamming the Teams Client.  
+
+   ```javascript
+   const firstChunk = 'Here is the first part of the information.';
+   const secondChunk = 'Continuing with more details...';
+
+   streamingResponse.queueTextChunk(firstChunk);
+
+   // Delay between chunks, minimum 1000ms
+   await new Promise(resolve => setTimeout(resolve, 1500));
+
+   streamingResponse.queueTextChunk(secondChunk);
+   ```
+
```

Comment:
Maybe add a note here about Adaptive Cards? In the original streaming doc I took a quick look and it shows you how to send the adaptive card but doesn't seem to explicitly say that it must be the last chunk, and streaming adaptive cards are not supported yet, no ETA. Unless I missed it? But I think some kind of reminder here would be good.

Reply:
Oh my bad, I did see that but I thought it was associated with AI section. Resolved!